README.md

```diff
@@ -34,6 +34,7 @@ app.use(limiter);
 ## Configuration
 
 * **expiry**: seconds - how long each rate limiting window exists for. Defaults to `60`.
+* **resetExpiryOnChange**: boolean - if the expiry time should be reset every time a key is incremented/decremented. This means that when the limit is reached and the user is given a 429 response, the expiry time serves as a cooldown. Defaults to `false`.
```

Comment:
I'd change:

```diff
- the expiry time serves as a cooldown. 
+ the rate limit window is extended.
```

Other than that, it looks good! The default `false` makes lots of sense too.